clients/lsp-json.el

```diff
@@ -98,7 +98,7 @@
                   (list callback))))
 
 (lsp-dependency 'vscode-json-languageserver
-                '(:system "vscode-json-language-server")
+                '(:system "vscode-json-languageserver")
```

Comment:
if the binary was renamed in some release we could support both adding two `:system`?

Reply:
@ericdallo Maybe, but I'm not sure. This is the first time I'm trying to use JSON language server and I got hit with this.

Also, I'm not able to find any CHANGELOG associated with the vscode project.

Reply:
I think we can go ahead and if anyone complain we can add back again the old system